src/sqlancer/postgres/PostgresToStringVisitor.java

```diff
@@ -331,9 +331,8 @@ public void visit(PostgresSimilarTo op) {
         visit(op.getString());
         sb.append(" SIMILAR TO ");
         visit(op.getSimilarTo());
-        if (op.getEscapeCharacter() != null) {
-            visit(op.getEscapeCharacter());
-        }
+        sb.append(" ESCAPE ");
```

Comment:
If it's optional, we don't want to always generate this, as we would reduce the search space.